### PR TITLE
NTTW2 info about lunch + sponsored social dinner

### PR DESCRIPTION
**Pull request by JeromeMartinez**

### CODE REVIEW

**Comment by ablwr** [comment]
🌭 🌭 🌭 🌭 🌭 🌭 🌭 🌭 🌭

File: notimetowait2.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ Event Hashtag: **#notimetowait**
 
 Hosted by the [Österreichisches Filmmuseum - Austrian Film Museum](https://www.filmmuseum.at) and [MediaArea.net](https://mediaarea.net)
 
+Locally organized by [AV-RD](http://www.av-rd.com) and social dinner sponsored by [ANYFER](http://www.anyfer.com)
```
Comment (dericed) [inline notimetowait2.md]: but when?
Also from the phrasing it's unclear what is locally organized by AV-RD: the entire symposium is the dinner.

File: notimetowait2.md [hunk 2]
```diff
@@ -44,6 +46,8 @@ Thursday, November 9, 2017 and Friday, November 10, 2017
 Opening: 09:00  
 Closing: 17:00 (sharp)  
 
+The 2 day symposium includes drinks and (cold) lunch on site, and a social dinner is planned on Thursday
```
Comment (dericed) [inline notimetowait2.md]: OK, maybe the Thursday here it related to the dinner mentioned on line 17, but they are far apart. Is `(cold)` necessary?

File: notimetowait2.md (hunk 1 above)
Comment (JeromeMartinez) [reply notimetowait2.md]: I didn't want to put the "technical" (i.e. date) details there, only listing involved people.
I don't see the unclear part, there is something before the 'and' and something after.
Suggestion?

File: notimetowait2.md (hunk 2 above)
Comment (JeromeMartinez) [reply notimetowait2.md]: 'cold' is not necessary, only for debate. I can remove it.
This is sae dinner, only technical part (the date). Suggestion?

**Review by dericed — APPROVED** [review]

**Pull request merged**